.goreleaser.yaml

```diff
@@ -6,43 +15,66 @@ builds:
       - linux
       - windows
     goarch:
-      - 386
+      - "386"
       - amd64
       - arm64
+    ldflags:
+      - -s
+      - -w
+      - -X 'main.version={{.Version}}'
+    ignore:
+      - goos: darwin
+        goarch: "386"
     env:
-      - CGO_ENABLED=0
+      - TOOLCHAIN_BASE=
```

Comment:
A comment explaining what this is trying to achieve (and why we need to do it manually) would be great 👍